docs/guides/references/configuration.mdx

```diff
@@ -205,16 +205,17 @@ object:
+| `indexHtmlFile`                | `cypress/support/component-index.html`    | This is where Cypress renders your components and allows you to add in global assets, such as styles, fonts, and external scripts.                                                                                                                                                        |
+| `setupNodeEvents`              | `null`                                    | Function in which node events can be registered and config can be modified. Takes the place of the (removed) plugins file. [Please read the notes for examples on using this.](#setupNodeEvents)                                                                                          |
+| `supportFile`                  | `cypress/support/component.js`            | Path to file to load before spec files load. This file is compiled and bundled. (Pass `false` to disable)                                                                                                                                                                                 |
+| `specPattern`                  | <code>\*\*/\*.cy.\{js,jsx,ts,tsx\}</code> | A glob pattern String or Array of glob pattern Strings of the spec files to load. <br /><br />Note that any files found matching the `e2e.specPattern` value will be automatically **excluded.**                                                                                          |
+| `excludeSpecPattern`           | `['/snapshots/*', '/image_snapshots/*']`  | A String or Array of glob patterns used to ignore spec files that would otherwise be shown in your list of specs. [Please read the notes on using this.](#excludeSpecPattern)                                                                                                             |
+| `experimentalSingleTabRunMode` | `false`                                   | Run all specs in a single tab, instead of creating a new tab per spec. This can improve run mode performance, but can impact spec isolation and reliability on large test suites. This experiment currently only applies to Component Testing.                                            |
+| `slowTestThreshold`            | `250`                                     | Time, in milliseconds, to consider a test "slow" during `cypress run`. A slow test will display in orange text in the default reporter.                                                                                                                                                   |
+| `devServerPublicPathRoute`     | `/__cypress/src`                          | The public path of the dev server in use. We recommend using the Cypress default, but may need to be changed. See [Framework Configuration](/guides/component-testing/component-framework-configuration) or dev server specific documentation (in particular `@cypress/vite-dev-server`). |
```

Comment:
I would just remove the second sentence since we aren't stating why it may need to be changed.

```suggestion
| `devServerPublicPathRoute`     | `/__cypress/src`                          | The public path of the dev server in use. See [Framework Configuration](/guides/component-testing/component-framework-configuration) or dev server specific documentation (in particular `@cypress/vite-dev-server`). |
```
Also, why "in particular" the `vite-dev-server`? Is there somewhere we can link the users for the dev server documentation?

Reply:
I mean we definitely want to recommend the default unless you need to specify it. We could link the readme in the repository? https://github.com/cypress-io/cypress/blob/develop/npm/vite-dev-server/README.md#compatibility

Reply:
It just feels awkward to me since we aren't saying what issues it would cause or when the user should change the value. I would argue that we recommend the default for all of our config unless the user needs to specify it. If we want to discourage setting the value, I would say something like "Use caution overriding the default value as it can have unintended consequences."

Reply:
OK I reworded in a0e6c6bc6. How does that look?